src/ViewMarc.js

```diff
@@ -29,6 +29,7 @@ class ViewMarc extends React.Component {
     } = this.props;
 
     if (!marcRecord) {
+      this.setState({ marcRecord: null });
```

Comment:
Not sure this line is needed since `marcRecord` is already set to null [here](https://github.com/folio-org/ui-inventory/pull/687/files#diff-a7ab07b300a26c67bd1523c3d03879f3R22).

Reply:
Done